server/server.js

```diff
@@ -66,4 +78,64 @@ app.delete('/api/lobby', (req, res) => {
   res.status(200).send('games deleted!');
 });
 
+app.get('/api/game/:id', (req, res) => {
+  let g = {}
+
+  for (let game of games) {
```

Comment:
Kanske skulle kunna använda Array.prototype.filter här istället, minska koden lite?

Reply:
anledningen bakom for of är att vi endast vill returnera ett objekt, i detta fallet med filter kommer vi alltid returnera en array med endast ett objekt, bör man ändå göra det?

Reply:
Tänker man kunde skrivit,
`const g = games.filter((game) => game.id === req.params.id)[0] || {}`
Dvs, filtrera games, ta ut första resultatet - finns det inget sådant(och således undefined) så defaulta till tomt objekt @osynkad

Reply:
ah okej, är med på det!
thx! 👍